petabtests/cases/v2.0.0/sbml/0023/_experiments.tsv

```diff
@@ -0,0 +1,2 @@
+experimentId	time	conditionId
+experiment1	-inf	
```

Comment:
How does simulation work here? The measurements are for `t=0` and `t=inf`, but there is only a preeq condition, no condition after preeq? So the preeq condition is also used for the `t=inf` measurement? Seems like it should be an invalid problem...

Reply:
No condition at t=0 means just not changing anything after preequilibration, yes. I don't think that's illegal. (If you think it is, or should be, let's discuss that at https://github.com/PEtab-dev/PEtab).
If there are no events that might interfere, then this example could be simplified by dropping pre-equilibration and just changing both measurements to t=inf. If not, then there might be subtle differences (depending on https://github.com/PEtab-dev/PEtab/pull/635 and https://github.com/PEtab-dev/PEtab/issues/642).

Reply:
Hm you're right. It looks like something a user might do if they don't understand steady-states in PEtab, but no need to exclude.